### PR TITLE
Create base visualization template and tooling command

### PR DESCRIPTION
**Pull request by michplunkett**
## What does this PR do?
It creates the template by which we'll create all future data visualizations, standardizes the font to `HCo_Verlag`, and creates the command that will copy the template to a new story.

## Testing
- [x] Manually tested
```
michaelp@MacBook-Air-3 chicagomaroon.github.io % npm run create-visual --year=2023 --story=new-story 

> chicagomaroon.github.io@1.0.0 create-visual
> mkdir -p ./$npm_config_year/$npm_config_story && cp ./template_visual/* ./$npm_config_year/$npm_config_story

michaelp@MacBook-Air-3 chicagomaroon.github.io % npm run process-visual --year=2023 --story=new-story

> chicagomaroon.github.io@1.0.0 process-visual
> npx htmlprocessor ./$npm_config_year/$npm_config_story/index.html -o ./$npm_config_year/$npm_config_story/index.proc.html && npx minify ./$npm_config_year/$npm_config_story/index.proc.html > ./$npm_config_year/$npm_config_story/$npm_config_story.min.html && rm ./$npm_config_year/$npm_config_story/index.proc.html
File "./2023/new-story/index.proc.html" created.
michaelp@MacBook-Air-3 chicagomaroon.github.io % 
```

### CODE REVIEW

File: 2016/crosswords/lib/jScrollPane.js [hunk 1]
```diff
@@ -562,7 +562,6 @@
                     var ceaseAnimation = function () {
                         if (_animateToInterval) {
                             clearInterval(_animateToInterval);
-                            delete _animateToPosition;
```
Comment (michplunkett) [inline 2016/crosswords/lib/jScrollPane.js:565]: Linting issue.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# chicagomaroon.github.io
+# The Chicago Maroon's Data Visualizations
```
Comment (michplunkett) [inline README.md:1]: Could you read over the text in this file, @ywlu026?

File: package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
     "description": "Feature stories by The Chicago Maroon grouped by year into corresponding folders.",
     "main": "index.js",
     "scripts": {
+        "create-visual": "mkdir -p ./$npm_config_year/$npm_config_story && cp ./template_visual/* ./$npm_config_year/$npm_config_story",
```
Comment (michplunkett) [inline package.json:7]: Here's the command for creating a new visual.

File: README.md [hunk 4]
```diff
@@ -16,17 +16,18 @@ Feature stories by The Chicago Maroon grouped by year into corresponding folders
...
 
 ## Build process for a visualization
 1. Go into the main directory `./chicagomaroon.github.io` and run the command `npm install`
-2. Create your HTML with the visualization inside its respective year and feature folder.
-    - If it is the year `2023` and your story is called `Example Visual` then go to the year folder `2023` and your story's folder `example-visual`.
-    - If either of those folders do not exist yet, please create them. Please use [kebab](https://www.freecodecamp.org/news/programming-naming-conventions-explained#what-is-kebab-case) case when creating folder names.
-3. Create your visual in a file called `index.html`.
+2. Create your HTML with the visualization inside its respective year and story folder via the `npm run create-visual` command.
+    - If it is the year `2023` and your story is called `Example Visual` then go to the year folder `2023` and your story's folder `example-visual` in [kebab-case]((https://www.freecodecamp.org/news/programming-naming-conventions-explained#what-is-kebab-case).
```
Comment (michplunkett) [inline README.md]: Need to fix link here and folder reference above.

**Comment by michplunkett** [comment]
I need to move the font file to the static file directory, there's no need to write it over and over.

**Review by ywlu026 — APPROVED** [review]

**Pull request merged**